common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3899,6 +3899,12 @@
     },
     "sqlState" : "42605"
   },
+  "XML_ROW_TAG_OPTION_REQUIRED" : {
```

Comment:
How about `XML_ROW_TAG_MISSING`?

Reply:
Yep, I agree. The error class name should point out a problem, possible solutions should be in `message`.